tooling/enforcer-rules/allow-findbugs.xsl

```diff
@@ -21,6 +21,8 @@
 
  <xsl:output omit-xml-declaration="yes"/>
 
+    <xsl:param name="findbugs" select="'${allow-findbugs}'"/>
```

Comment:
@ppalaga I introduce a param here to copy the `findbugs` rules if it is not ture. Not sure if it would impact the `cq-plugin-maven` with the FlattenMojo?